statsmodels/genmod/cov_struct.py

```diff
@@ -927,9 +927,22 @@ class Equivalence(CovStruct):
       one triangle of each covariance matrix should be included.
       Positions where j1 and j2 have the same value are variance
       parameters.
+    labels : array-like
+      An array of labels such that every distinct pair of labels
+      defines an equivalence class.  Either `labels` or `pairs` must
+      be provided.  When the two labels in a pair are equal two
+      equivalence classes are defined: one for the diagonal elements
+      (corresponding to variances) and one for the off-diagonal
+      elements (corresponding to covariances).
+    as_cor : boolean
```

Comment:
`as_cor` doesn't sound like a good name (I haven't looked enough to suggest an alternative.)